sig-scalability/slos/slos.md

```diff
@@ -11,26 +11,29 @@ in these areas.
 
 ## What do we require from SLIs/SLOs?
 
-We are going to define more SLIs and SLOs based on the most important indicators
-in the system.
+We are in the process of extending the number of SLIs ([Service Level Indicators])
+and SLOs ([Service Level Objectives]) built on top of these SLIs to cover more areas
+of the system and user expectations.
 
 Our SLOs need to have the following properties:
 - <b> They need to be testable </b> <br/>
-  That means that we need to have a benchmark to measure if it's met.
+  Ideally, they should be measurable in the running cluster, but if
```

Comment:
Could you clarify what is not measurable, SLI or SLO?

Reply:
Both :)
But it's mostly about the SLI. SLO is generally defined as some aggregation of SLI <= X, which is generally quite easy to compute. SLI is what might be hard to measure in all running clusters.

Reply:
Done.

Reply:
> But it's mostly about the SLI. SLO is generally defined as some aggregation of SLI <= X, which is generally quite easy to compute. SLI is what might be hard to measure in all running clusters.

Yes, that's why I was asking :)

Still there's one more thing that I'm not sure about. How do you infer "In other words, not every SLO may be translantable to SLA" from the earlier lines?

Reply:
If it's not possible to measure it on every running cluster (i.e. if we only have benchmark), you obviously can't have SLA for it (because you're not able to measure it for user clusters).

Reply:
Then I wouldn't say "In other words", but rather "So that means" (as it's not an equivalence, but an implication).

Reply:
done.